incubator/command-dump/src/test/java/io/aklivity/zilla/runtime/command/dump/internal/TestDumpCommand.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.command.dump.internal;
+
+import static org.pcap4j.core.PcapHandle.TimestampPrecision.NANO;
+
+import java.io.EOFException;
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.pcap4j.core.NotOpenException;
+import org.pcap4j.core.PcapHandle;
+import org.pcap4j.core.PcapNativeException;
+import org.pcap4j.core.Pcaps;
+import org.pcap4j.packet.Packet;
+
+import com.google.common.io.Files;
+
+import io.aklivity.zilla.runtime.command.dump.internal.airline.DumpStreamsCommand;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.spy.RingBufferSpy;
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.EngineConfiguration;
+
+public class TestDumpCommand
+{
+
+    @TempDir
+    File tempDir;
+
+    @Test
+    public void testDump() throws PcapNativeException, NotOpenException, IOException, TimeoutException
+    {
+        Properties properties = new Properties();
+        properties.put("zilla.engine.directory", "src/test/resources/zilla_test_stream");
```

Comment:
Please move all resources used by the test to the parallel package path for this test class, but (deeper) under `src/test/resources/...`.

Suggest using `src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/engine` and `src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/expected.pcap` or similar.